### PR TITLE
chore: add translations for dialogs

### PR DESCRIPTION
**Pull request by timarney**
# Summary | Résumé

Fixes: https://github.com/cds-snc/platform-forms-client/issues/3996

Adds translation strings for 

1) Deleting a page from the Tree View

2) Confirm moving a page when custom rules are set

For this go to logic page --- add some custom rules to a page
Go to Tree View and attempt to move that page

### CODE REVIEW

File: i18n/translations/en/form-builder.json [hunk 1]
```diff
@@ -975,6 +975,18 @@
...
+      "title": "Delete this group?",
+      "description": "<span className='inline-block mb-4'><strong>Questions will also be deleted</strong><br />Elements within the group will also be deleted.</span><span className='inline-block mb-4'><strong>Move questions </strong><br />Move all elements that you would like to keep to another group.</span>",
+      "cancel": "Cancel",
+      "delete": "Delete group and elements"
+    },
+    "confirmMoveGroup": {
+      "title": "Rewrite rules?",
+      "description": " It seems the item(s) you are moving have one or more custom rules. Would you like to overwrite those rules with autoflow?",
+      "cancel": "No, move without changing rules",
+      "confirm": "Yes, autoflow"
```
Comment (anikbrazeau) [inline i18n/translations/en/form-builder.json:989]:
```suggestion
    },
    "confirmDeleteGroup": {
      "title": "Delete this page?",
      "description": "<span className='inline-block mb-4'><strong>Page content will also be deleted</strong><br />All form elements on this page will be deleted.</span><span className='inline-block mb-4'><strong>You can move content before deleting</strong><br />Move all form elements that you would like to keep to another page.</span>",
      "cancel": "Cancel",
      "delete": "Delete page and its contents"
    },
    "confirmMoveGroup": {
      "title": "Overwrite rules?",
      "description": " It seems the items you are moving may have custom rules applied. Would you like to overwrite those rules and apply linear autoflow?",
      "cancel": "No, move without changing rules",
      "confirm": "Yes, apply linear autoflow"
```

File: i18n/translations/fr/form-builder.json [hunk 2]
```diff
@@ -975,6 +975,18 @@
...
+      "title": "Delete this group? [FR]",
+      "description": "<p><strong>Questions will also be deleted</strong><br />Elements within the group will also be deleted.</p><p><strong>Move questions </strong><br />Move all elements that you would like to keep to another group.</p> [FR]",
+      "cancel": "Cancel [FR]",
+      "delete": "Delete group and elements [FR]"
+    },
+    "confirmMoveGroup": {
+      "title": "Rewrite rules? [FR]",
+      "description": " It seems the item(s) you are moving have one or more custom rules. Would you like to overwrite those rules with autoflow? [FR]",
+      "cancel": "No, move without changing rules [FR]",
+      "confirm": "Yes, autoflow [FR]"
```
Comment (anikbrazeau) [inline i18n/translations/fr/form-builder.json:989]:
```suggestion
    "confirmDeleteGroup": {
      "title": "Supprimer cette page?",
      "description": "<p><strong>Le contenu de la page sera supprimé</strong><br />Tous les éléments de cette page seront également supprimés.</p><p><strong>Vous pouvez déplacer le contenu </strong><br />Déplacez tous les éléments de formulaire que vous souhaitez conserver vers une autre page.</p>",
      "cancel": "Annuler",
      "delete": "Supprimer la page et son contenu"
    },
    "confirmMoveGroup": {
      "title": "Remplacer les règles",
      "description": " Il semble que les éléments déplacés comportent des règles conditionnelles. Souhaitez-vous remplacer ces règles par un flux automatique linéaire?",
      "cancel": "Non, déplacer sans modifier les règles",
      "confirm": "Oui, appliquer un flux automatique"
```

File: i18n/translations/fr/form-builder.json [hunk 3]
```diff
@@ -975,6 +975,18 @@
...
+      "title": "Supprimer cette page?",
+      "description": "<p><strong>Le contenu de la page sera supprimé</strong><br />Tous les éléments de cette page seront également supprimés.</p><p><strong>Vous pouvez déplacer le contenu </strong><br />Déplacez tous les éléments de formulaire que vous souhaitez conserver vers une autre page.</p>",
+      "cancel": "Annuler",
+      "delete": "Supprimer la page et son contenu"
+    },
+    "confirmMoveGroup": {
+      "title": "Remplacer les règles",
+      "description": " Il semble que les éléments déplacés comportent des règles conditionnelles. Souhaitez-vous remplacer ces règles par un flux automatique linéaire?",
+      "cancel": "Non, déplacer sans modifier les règles",
+      "confirm": "Oui, appliquer un flux automatique"
```
Comment (anikbrazeau) [inline i18n/translations/fr/form-builder.json:989]:
```suggestion
    },
    "confirmDeleteGroup": {
      "title": "Supprimer cette page?",
      "description": "<p><strong>Le contenu de la page sera également supprimé</strong><br />Tous les éléments de cette page seront supprimés.</p><p><strong>Vous pouvez déplacer le contenu avant de supprimer</strong><br />Déplacez tous les éléments de formulaire que vous souhaitez conserver vers une autre page.</p>",
      "cancel": "Annuler",
      "delete": "Supprimer la page et son contenu"
    },
    "confirmMoveGroup": {
      "title": "Remplacer les règles?",
      "description": " Il semble que des éléments déplacés comportent des règles conditionnelles. Souhaitez-vous remplacer ces règles par un flux automatique linéaire?",
      "cancel": "Non, déplacer sans modifier les règles",
      "confirm": "Oui, appliquer un flux automatique"
```

File: i18n/translations/en/form-builder.json [hunk 4]
```diff
@@ -975,6 +975,18 @@
...
+    },
+    "confirmDeleteGroup": {
+      "title": "Delete this page?",
+      "description": "<span className='inline-block mb-4'><strong>Page content will also be deleted</strong><br />All form elements on this page will be deleted.</span><span className='inline-block mb-4'><strong>You can move content before deleting</strong><br />Move all form elements that you would like to keep to another page.</span>",
+      "cancel": "Cancel",
+      "delete": "Delete page and its contents"
+    },
+    "confirmMoveGroup": {
+      "title": "Overwrite rules?",
+      "description": " It seems the items you are moving may have custom rules applied. Would you like to overwrite those rules and apply linear autoflow?",
```
Comment (anikbrazeau) [inline i18n/translations/en/form-builder.json:987]:
```suggestion
      "description": " It seems the items you are moving have conditional rules applied. Would you like to overwrite those rules and apply linear autoflow?",
```

**Review by thiessenp-cds — APPROVED** [review]
LGTM :)

**Pull request merged**